src/subdomains/generic/user/models/user-data/user-data.service.ts

```diff
@@ -396,7 +402,7 @@ export class UserDataService {
         nationality: { id: dto.nationality?.id ?? userData.nationality?.id },
         identDocumentId: dto.identDocumentId ?? userData.identDocumentId,
       });
-      if (existing && userData.id !== existing.id)
+      if (existing?.identDocumentId && userData.id !== existing.id)
```

Comment:
I think the db request would still find existing entities with an identDocumentId if `dto.identDocumentId` and `userData.identDocumentId` are undefined. 
I would either change the if line to `if(existing && (dto.identDocumentId || userData.identDocumentId) && userData.id !== existing.id)` or we have to add an `?? IsNull()` at the end of the identDocumentId where request